h5netcdf/core.py

```diff
@@ -764,8 +777,49 @@ def _resize_variables(self, dimension, size, recurse=True):
 
 class File(Group):
     def __init__(
-        self, path, mode=None, invalid_netcdf=False, phony_dims=None, **kwargs
+        self, path, mode=None, invalid_netcdf=False, phony_dims=None,
+        *, track_order=True, **kwargs
     ):
+        """NetCDF4 file constructor.
+
+        Parameters
+        ----------
+        path: path-like
+            Location of the netCDF4 file to be accessed.
+
+        mode: "r", "r+", "a", "w"
+            A valid file access mode. See
+
+        invalid_netcdf: bool
+            Allow writing netCDF4 with data types and attributes that would
+            otherwise not generate netCDF4 files that can be read by other
+            applications. See
+            https://github.com/h5netcdf/h5netcdf#invalid-netcdf-files
+            for more details.
+
+        phony_dims: 'sort', 'access'
+            See:
+            https://github.com/h5netcdf/h5netcdf#datasets-with-missing-dimension-scales
+
+        track_order: bool
+            Enable order tracking of group creation within the HDF5 file.
+            This is required by the netCDF4 file format, though it seems
+            optional and may limit performance. In versions 0.12.0 and earlier
+            the default value was ``False``.
+            See https://github.com/h5netcdf/h5netcdf/issues/128 for
+            more information.
+
+        **kwargs:
+            Additional keyword arguments to be passed to the ``h5py.File``
+            constructor.
+        """
+        # 2022/01/09
```

Comment:
We might just add these information to the issue and reference the issue.

Reply:
To be honest, these two sentences were to act like a short summary.
https://github.com/h5netcdf/h5netcdf/issues/128#issuecomment-1008396755

Reply:
Let's leave it as is for now. Maybe good enough, if people just see it in the code.